migrate_pim/apply_patch/apply_patch_ce.rst

```diff
@@ -0,0 +1,47 @@
+Guide to apply a patch for Community Edition
+=============================================
+
+Run the composer update command:
+
+.. code-block:: bash
+
+    composer --prefer-dist update
+
+Be aware that your composer.json won't be updated and some dependencies might be missing or from an outdated version.
```

Comment:
In the last part: "might be missing or from an outdated version."
I would say: "missing or coming from ..."

Reply:
:+1: